Formula/c/certifi.rb

```diff
@@ -15,24 +15,37 @@ class Certifi < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "1f1fc985a1c89bd40c73b17e3dfbf5483cb0417c8d4d12e2be66158a503ab169"
   end
 
-  depends_on "python@3.11" => [:build, :test]
-  depends_on "python@3.12" => [:build, :test]
   depends_on "python@3.13" => [:build, :test]
+  depends_on "python@3.11" => :test
+  depends_on "python@3.12" => :test
   depends_on "ca-certificates"
 
   def pythons
     deps.map(&:to_formula).sort_by(&:version).filter { |f| f.name.start_with?("python@") }
   end
 
   def install
-    pythons.each do |python|
-      python_exe = python.opt_libexec/"bin/python"
-      system python_exe, "-m", "pip", "install", *std_pip_args(build_isolation: true), "."
+    # Ensure uniform bottles
+    inreplace "README.rst", "/usr/local", HOMEBREW_PREFIX
+
+    python = deps.find { |dep| dep.name.start_with?("python@") && dep.build? }
+                 .to_formula
+                 .opt_libexec/"bin/python"
+    system python, "-m", "pip", "install", *std_pip_args(build_isolation: true), "."
+
+    # Use brewed ca-certificates PEM file instead of the bundled copy
+    site_packages = Language::Python.site_packages(python)
+    rm prefix/site_packages/"certifi/cacert.pem"
+    (prefix/site_packages/"certifi").install_symlink Formula["ca-certificates"].pkgetc/"cert.pem" => "cacert.pem"
+
+    # Create symlinks so that the Python bindings can be used with alternative Python versions
+    python_versions = Formula.names
+                             .select { |name| name.start_with? "python@" }
+                             .map { |py| py.delete_prefix("python@") }
+    python_versions.each do |py_ver|
+      next if py_ver == Language::Python.major_minor_version(python).to_s
 
-      # Use brewed ca-certificates PEM file instead of the bundled copy
-      site_packages = Language::Python.site_packages("python#{python.version.major_minor}")
-      rm prefix/site_packages/"certifi/cacert.pem"
-      (prefix/site_packages/"certifi").install_symlink Formula["ca-certificates"].pkgetc/"cert.pem" => "cacert.pem"
+      (lib/"python#{py_ver}/site-packages").install_symlink (prefix/site_packages).children
```

Comment:
One thing to check on is cache files since the directory is symlinked. So `.pyc`/`__pycache__`/etc may be created inside `lib/python3.13/site-packages` for all versions.

A safer option could be symlinking files and recreating directory structure.